openapi/api_fast_login.template.yaml

```diff
@@ -72,3 +89,15 @@ components:
       $ref: "./generated/api_public.yaml#/definitions/AccessToken"
     ProblemJson:
       $ref: "https://raw.githubusercontent.com/pagopa/io-functions-commons/v28.6.0/openapi/definitions.yaml#/ProblemJson"
+    Nonce:
+      type: string
+      pattern: ^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$
+      description: UUID v4 formatted nonce as per RFC 4122
+      example: 870c6d89-a3c4-48b1-a796-cdacddaf94b4
+    GenerateNonceResponse:
+      type: object
+      properties:
+        nonce:
+          $ref: "#/components/schemas/Nonce"
+      required:
+        - nonce
```

Comment:
We could link the ones defined in [fast-login openapi](https://github.com/pagopa/io-functions-fast-login/blob/1b47d311847b7f63dbfd97886122f4b2e94b79b4/api/internal.yaml) and let the generation step to resolve them.

Reply:
done in https://github.com/pagopa/io-backend/pull/1069/commits/ebe95faee4abff7df275f3a484a410e953bbec4f